sphinx/users/comlinetools/precompressed.rst

```diff
@@ -0,0 +1,52 @@
+"Precompressed" tiles
+=====================
+
+Some formats allow reading and/or writing "precompressed" tiles.
+This feature is primarily intended for whole slide images, each of which may have thousands of tiles.
+Data representing these tiles may be compressed, either lossily or losslessly, depending on the
+file format, imaging modality, or acquisition configuration.
+
+When reading, this allows for retrieving compressed bytes that are as close as possible
+to what is actually stored in the file. Decompression (if necessary) is the responsibility
+of the consumer.
+
+When writing, compressed bytes instead of raw bytes can be provided to the writer,
+such that the writer will just save the provided compressed bytes instead of recompressing.
+
+This feature is primarily exposed in the :program:`bfconvert` command, using the `-precompressed` option.
+When the `-precompressed` option is used, best results are obtained by:
+
+* using the `-noflat` option
+* _not_ using the `-compression` option (this can work, but may force tile recompression)
+* _not_ using the `-tilex` or `-tiley` options (this can work, but may force tile recompression)
+
+There are several advantages to using the "precompressed" feature for formats that support it:
+
+* faster tile read and write times, as no tile compression/decompression needs to be performed
+* no change in compression quality
+* very little change in file sizes between input and output data
+
+There are also a few disadvantages:
+
+* tile sizes cannot be changed during conversion
+* compression type cannot be changed during conversion
+* input and output format must support same compression type and tile size
+* not all formats supported by Bio-Formats currently allow reading/writing precompressed tiles
+* tools other than :program:`bfconvert` do not currently make use of the precompressed tile API
+
+Formats that support precompressed tile reading:
```

Comment:
Assuming additional formats get added to this list mid-term, https://bio-formats.readthedocs.io/en/latest/formats/index.html would probably the natural section to keep a reference of all readers/writers supporting precompressed tile reading/writing. If a developer page gets introduced about precompressed tile reading/writing, this would also be more natural as a link rather than adding a reference to a subsection of a command-line utility page. 

Reviewing the few available options:
- https://bio-formats.readthedocs.io/en/latest/supported-formats.html includes an export column which describes the writing status (although this is definitely not obvious). Adding 2 more columns to this already heavy table would probably be detrimental to the overall readability
- https://bio-formats.readthedocs.io/en/latest/formats/options.html describes additional reader/writing options. This was introduced with regards to `DynamicMetadataOptions` but could include additional reader/writer specificity
- a new page listing reader/writers with pre-compressed tile support

Reply:
I tried to split into two pages - one under `/formats/` that has the high-level summary and list of supported readers/writers, and the original one under `/users/comlinetools/` that's more bfconvert-specific. Open to other thoughts though if that doesn't make sense.